metron-platform/metron-storm-kafka/src/main/java/org/apache/metron/storm/kafka/flux/SimpleStormKafkaBuilder.java

```diff
@@ -202,31 +240,4 @@ public String getTopic() {
     return new StormKafkaSpout<>(builder);
   }
 
-  private static Map<String, Object> modifyKafkaProps(Map<String, Object> props, String zkQuorum) {
-    try {
-      if(!props.containsKey(KafkaSpoutConfig.Consumer.BOOTSTRAP_SERVERS)) {
-        //this isn't a putIfAbsent because I only want to pull the brokers from zk if it's absent.
-        List<String> brokers = KafkaUtils.INSTANCE.getBrokersFromZookeeper(zkQuorum);
-        props.put(KafkaSpoutConfig.Consumer.BOOTSTRAP_SERVERS, Joiner.on(",").join(brokers));
-      }
-      props.putIfAbsent(KafkaSpoutConfig.Consumer.KEY_DESERIALIZER, ByteArrayDeserializer.class.getName());
-      props.putIfAbsent(KafkaSpoutConfig.Consumer.VALUE_DESERIALIZER, ByteArrayDeserializer.class.getName());
-
-    } catch (Exception e) {
-      throw new IllegalStateException("Unable to retrieve brokers from zookeeper: " + e.getMessage(), e);
-    }
-    return props;
-  }
-
-  private static <K,V> KafkaSpoutTuplesBuilder<K, V> createTuplesBuilder(List<String> config, String topic) {
-    TupleBuilder<K, V> tb =  new TupleBuilder<K, V>(topic, FieldsConfiguration.toList(config));
-    return new KafkaSpoutTuplesBuilderNamedTopics.Builder<>(tb).build();
-  }
-
-
-  private static KafkaSpoutStreams createStreams(List<String> config, String topic) {
-    final Fields fields = FieldsConfiguration.getFields(FieldsConfiguration.toList(config));
-    return new KafkaSpoutStreamsNamedTopics.Builder(fields, STREAM, new String[] { topic} ).build();
```

Comment:
STREAM isn't used anymore, right? Can we kill the field (and no, I can't comment on the field because it's not technically part of the diff, sigh).